projects/cn0561/src/cn0561.c

```diff
@@ -330,6 +334,29 @@ int main()
 		}
 	}
 
+#ifdef CN0561_REG_DUMP
```

Comment:
this is good but i think it'd be more useful if you'd put it in a function so that it could be used from anywhere at run-time, not just this one place

Reply:
Should I add this in the ad713x driver then?

Reply:
My opinion is that something like that is useful in the driver, others may differ... I think there are some no-OS drivers which have a regmap dump API.

Reply:
in most cases it's a simple for loop with a register read call, so no dedicated function is needed

but such a register dump function it's useful when there are gaps (like for this device) in the regmap